e2e2/test/cases/neuron/manifests/bert-train.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: bert-mpi-training
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    backoffLimit: 20
+    cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - image: public.ecr.aws/o5d5x8n6/weicongw:neuron
+            imagePullPolicy: Always
+            name: bert-training-launcher
+            env:
+            - name: NEURON_CORES
+              value: "32"
+            command:
+            - /opt/amazon/openmpi/bin/mpirun
+            - --allow-run-as-root
+            - -np
+            - "{{.WorkerNodeCount}}"
+            - -bind-to
+            - none
+            - -map-by
+            - slot
+            - -x
+            - PATH
+            - -x
+            - LD_LIBRARY_PATH
+            - -x
+            - NEURON_CORES
+            - bash
+            - ./e2e-test/bert-train.sh
+    Worker:
+      replicas: 2
```

Comment:
This needs to be set to 4 to be compliant with what will be ran in the AMI release pipeline, or at least parameterized.

Reply:
I forgot to update this, this should be `{{.WorkerNodeCount}}`